tests/test_docker_latex.py

```diff
@@ -0,0 +1,83 @@
+import unittest
+import os
+import subprocess
+import shutil
+from quizgen.latex import compile
+from quizgen.quiz import Quiz
+from quizgen.pdf import make_with_path
+from quizgen.util.json import loads, dump_path
+
+class DockerTestBase(unittest.TestCase):
+    DOCKER_IMAGE = "edulinq-quizgen-tex:latest"
+
+    def setUp(self):
+        self.test_dir = os.path.join("tests", "test_output")
+        os.makedirs(self.test_dir, exist_ok=True)
+
+    def tearDown(self):
+        if os.path.exists(self.test_dir):
+            shutil.rmtree(self.test_dir)
```

Comment:
I don't think these are too necessary, we already have the utils in quizgen.util.dirent.

Maybe instead you can do quizgen.latex.set_pdflatex_use_docker() instead.